cves/CVE-2016-7905.yml

```diff
@@ -90,22 +93,29 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: 2679ad4773aa356e7c3da5c68bc81f02a194617f
-  note: ''
-- commit: ''
-  note: ''
+  note: '3.2 vulnerability fix'
+- commit: 622ccbd8ab894e3ac6cdf607e3d4f39e406786e9
+  note: '3.1.4 vulnerability fix'
+- commit: 8c43f320574d201fe1b696b133c08368f5f18508
+  note: '3.0.4 vulnerability fix'
+- commit: 239f75d6c3dfbe4def80a12913d5737dd5a5bbcc
+  note: 2.8.9 vulnerability fix'
+- commit: 45f5e17aa43a4cebed77d8e9781447ad29a0ed61
+  note: '2.4.14 vulnerability fix'
+
 vccs:
 - commit: 4641ae352ec587355764ffd5c43dd0d0ebd47654
-  note: Identified by archeogit,
+  note: Identified by archeogit, Yes deal with the vulnerability with other too
 - commit: 3c90cc2ef21a928a7c11cf5a4583169025cdf847
-  note: Identified by archeogit,
+  note: Identified by archeogit, Yes deal with the vulnerability with other too
 - commit: b76234c00c7cac154dac50f4993e4372e47e3718
-  note: Identified by archeogit,
+  note: Identified by archeogit, Yes deal with the vulnerability with other too
 - commit: 9200514ad8717c63f82101dc394f4378854325bf
-  note: Identified by archeogit,
+  note: Identified by archeogit, Yes deal with the vulnerability with other too
```

Comment:
Are all of these commits that introduced the vulnerability? is there just one that actually introduced the vulnerability? Perhaps better notes would make it easier as a consumer to understand which each commit it doing.